core/controlplane/config/templates/cluster.yaml

```diff
@@ -998,6 +1004,12 @@ worker:
 #  tag: 1.1.2
 #  rktPullDocker: false
 
+# kiam image repository to use.
+kiamImage:
```

Comment:
Similar to https://github.com/kubernetes-incubator/kube-aws/pull/1002, I think having all the defaults in a single place is beneficial. 2.7 here now differs from 2.6 in the code, I'm not sure if it'd be better to remove the code level default or now.

Reply:
I'm ok with removing all the code level defaults, as long as missing values in cluster.yaml doesn't make kube-aws fail.
The reasoning is that probably users want to avoid including e.g. kiam related settings at all in cluster.yaml when one doesn't enable kiam.

Reply:
I agree some users want a minimal config file. We could always put all ththe defaults in on place in the code then generate this.